Classes/Settings/SettingsAccountsViewController.swift

```diff
@@ -113,6 +113,7 @@ final class SettingsAccountsViewController: UITableViewController, GitHubSession
 
     func didFocus(manager: GitHubSessionManager, userSession: GitHubUserSession, dismiss: Bool) {
         updateUserSessions()
+        ShortcutHandler.configure(application: UIApplication.shared, sessionManager: sessionManager)
```

Comment:
Wont the accounts not update if you switch using the 3d touch shortcut? Why not just add the app delegate as a listener and update in one place?

Reply:
I am not understanding right your suggestion 😕  @rnystrom .. 
I would do it like this:  

In AppDelegate
```swift
func application(_ application: UIApplication, performActionFor shortcutItem: UIApplicationShortcutItem, completionHandler: @escaping (Bool) -> Void) {
       ...
         ShortcutHandler.configure(
            application: UIApplication.shared,
            sessionManager: sessionManager)
    }
```

and 

In Settings

```swift
func application(_ application: UIApplication, performActionFor shortcutItem: UIApplicationShortcutItem, completionHandler: @escaping (Bool) -> Void) {
       ...
         ShortcutHandler.configure(
            application: UIApplication.shared,
            sessionManager: sessionManager)
    }
```

Reply:
In [didFinishLaunching](https://github.com/GitHawkApp/GitHawk/blob/master/Classes/Systems/AppDelegate.swift#L32) add

```swift
sessionManager.addListener(listener: self)
```

Conform to the required protocol, then in the `didFocus(...)` function configure the shortcuts.